receiver/dockerstatsreceiver/integration_test.go

```diff
@@ -31,7 +31,7 @@ import (
 	"github.com/testcontainers/testcontainers-go/wait"
 	"go.opentelemetry.io/collector/component"
 	"go.opentelemetry.io/collector/consumer/consumertest"
-	"go.opentelemetry.io/collector/receiver"
+	rcvr "go.opentelemetry.io/collector/receiver"
```

Comment:
Why is this change needed? I don't see any `receiver` identifiers which it can conflict with

Reply:
There is a type `receiver` in the receiver.go.